.github/rpr.sh

```diff
@@ -4,9 +4,9 @@
 
 set -e # stop on error
 
-usage="$(basename "$0") pr -- rebase a Pull Request against current branch"
+usage="$(basename "$0") pr remote/branch -- rebase a Pull Request against a remote branch"
 
-if [ "$#" -ne 1 ]; then
+if [ "$#" -ne 2 ]; then
```

Comment:
Should we have a default value for the most common case ? (if `$2` is not present, use `upstream/master` for ex 👼 )